routing_model/src/scenario_tests.rs

```diff
@@ -17,6 +17,7 @@ use crate::utilities::{
     NodeState, ParsecVote, Proof, ProofRequest, ProofSource, Rpc, Section, SectionInfo,
 };
 
+#[cfg(test)]
```

Comment:
should it be in libs.rs since all this file should only compile for tests?

Reply:
Yes that makes sense :)